### DISCUSSION TITLE
Отсутствие кнопки "Копировать" в iframe

### DISCUSSION THREAD

**Discussion by a1ext0** [totumonline/totum-ru-issues-and-discussions#335] [Как это сделать?] [answered]
Приветствую, разработчики Totum! При обновлении на новую версию кнопка "Скопировать выделенные строки" переместилась из верхнего левого угла таблицы в правый нижний угол страницы. В обычных таблицых все работает хорошо. Но при этом в iframe теперь пропала возможность скопировать выделенные данные, в самом фрейме кнопки просто нет, а за пределами iframe она не активна

<img width="1437" alt="image" src="https://user-images.githubusercontent.com/54662036/225318832-3ae4481d-06ee-43ea-9b2d-13463538fd7c.png">

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Всем привет. В эту кнопку будет еще всякое разное добавлено. Что бы ее показать в `iframe` надо использовать параметр `hidedots: false` в `linkToTable` и `linkToDataTable`

Относительно скрытия копирования сверху идет дискуссия. Может еще и вернем обратно в зависимости от пожеланий пользователей.

**Reply by a1ext0** [reply]
Спасибо за ответ! Что касается пожеланий :)
то было бы здорово, чтобы или осталась кнопочка сверху, или по умолчанию `hidedots` был `false`. 

Обоснование: система Totum позиционируется как "продвинутые таблицы", поэтому пользователи часто используют функции копирования данных и нужно максимально упростить выдаваемый к копированию доступ для минимизации временных затрат разработчиков на Totum.

**Reply by ttm-alexeypolunin** [reply]
Вернем копировать наверх обратно в сл обновлении, но базово для `hidedots` оставим `true` тк в небольших `iframe` (таких большинство) он чет много места внизу занимает.

**Reply by ttm-alexeypolunin** [reply]
**готово**
